src/spring-cloud/HISTORY.rst

```diff
@@ -5,7 +5,7 @@ Release History
 
 0.2.1
 ++++++
-* Add command "az spring-cloud app log" to replace "az spring-cloud app log tail" for log streaming.
+* Add command "az spring-cloud app logs" to replace "az spring-cloud app log tail" for log streaming.

```

Comment:
I am confused with log and logs. Why sometimes it's single form and sometimes it's plural form?

Reply:
Hava synced about this offline. Thanks.